prep-javascript-variables/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    var fullName = 'Gabrielle Ramos';
+console.log(fullName);
+var numberOfPets = '1';
+console.log(numberOfPets);
+var likesVegetables = 'False';
+Boolean(likesVegetables);
+console.log(likesVegetables);
+var nothing = 'null';
```

Comment:
We'll want to leave out the quotes for numbers, booleans, and `null` so that the JavaScript engine doesn't interpret them as strings.